doc-content/enterprise-only/openshift/kieserver-additional-deploy-proc.adoc

```diff
@@ -31,7 +31,7 @@ In this command line:
 ** *KIE server controller user* (`KIE_SERVER_CONTROLLER_USER`): The controller user name for logging onto the {CENTRAL} that you configured.
 ** *KIE server controller password* (`KIE_SERVER_CONTROLLER_PWD`): The controller password for logging onto the {CENTRAL} that you configured.
 ** *KIE Server Keystore Secret Name* (`KIE_SERVER_HTTPS_SECRET`): The name of the secret for {KIE_SERVER}, as created in <<secrets-kie-create-proc_{context}>>.
-** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that the {KIE_SERVER} is to join.
+** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that this {KIE_SERVER} is to join.
```

Comment:
AB4

Reply:
yet another deleted file - these were from the old structure that got redone back in 7.3/7.4.